pages/science/reactors/cvodes.rst

```diff
@@ -0,0 +1,244 @@
+  user-provided time :math:`t_{\mathrm{new}}`. :math:`t_{\mathrm{new}}` is the absolute 
+  time from the initial time of the system. Although the user specifies the time when 
+  integration should stop, CVODES chooses the time step size as the network is integrated. 
+  Many of these internal CVODES time steps are usually required to reach 
+  :math:`t_{\mathrm{new}}`. As such, ``advance(t_new)`` preserves the accuracy of using 
+  ``step()`` but allows consistent time step spacing in the output data.
+
+- ``advance_to_steady_state(max_steps, residual_threshold, atol,
+  write_residuals)`` [Python interface only]: If the steady state solution of a
+  reactor network is of interest, this method can be used. Internally, the
+  steady state is approached by time stepping. The network is considered to be
+  at steady state if the feature-scaled residual of the state vector is below a
+  given threshold value (which by default is 10 times the time step ``rtol``).
+
+The ``advance(t_new)`` is typically used when consistent, regular, time steps are 
+required in the output data. This usually simplifies comparisons among many 
+simulation results at a single time point. However, some detail, for example, a 
+fast ignition process, might not be resolved in the output data if the user-provided 
+time step is not small enough.
+
+To avoid losing this detail, the
+`Reactor::setAdvanceLimit <{{% ct_docs doxygen/html/dc/d5e/classCantera_1_1Reactor.html#a9b630edc7d836e901886d7fd81134d9e %}}>`__
+method (C++) or the :py:func:`Reactor.set_advance_limit` method (Python) can be
+used to set the maximum amount that a specified solution component can change
+between output times. For an example of this feature's use, see the example
+`reactor1.py </examples/python/reactors/reactor1.py.html>`__. However, as a tradeoff, 
+the time step sizes in the output data are no longer guaranteed to be uniform.
+
+Even though Cantera comes pre-defined with typical parameters for tolerances
+and the maximum internal time step, the solver may not be correctly configured
+for the specific system. In this case the internal timestep solutions may not 
+converge towards a single value. To solve this problem, three parameters can be 
+tuned: The absolute time stepping tolerances, the relative time stepping tolerances, 
+and the maximum time step. A reduction of the latter value is particularly useful 
+when dealing with abrupt changes in the boundary conditions (for example, 
+opening/closing valves; see also the `IC engine example </examples/python/reactors
+/ic_engine.py.html>`__).
+
+How Does Cantera's Reactor Network Time Integration Feature Actually Work?
+==========================================================================
+
+There's a great description of the science behind Cantera's reactor network 
+simulation capabilities available on the Cantera website, 
+`here <https://cantera.org/science/reactors/reactors.html>`__. This section will go into more 
```

Comment:
I'm not sure we need to pat ourselves on the back about how *great* the other page is...